pennylane/pulse/rydberg_hamiltonian.py

```diff
@@ -236,18 +241,65 @@ def circuit_local(params):
         detuning,
     ]
 
-    drive_x_term = sum(qml.PauliX(wire) for wire in wires)
-    drive_y_term = sum(-qml.PauliY(wire) for wire in wires)
-    detuning_term = sum(qml.PauliZ(wire) for wire in wires)
+    drive_x_term = 0.5 * sum(qml.PauliX(wire) for wire in wires)
+    drive_y_term = -0.5 * sum(qml.PauliY(wire) for wire in wires)
+    detuning_term = -1.0 * sum(qml.PauliZ(wire) for wire in wires)
 
     observables = [drive_x_term, drive_y_term, detuning_term]
 
     # We convert the pulse data into a list of ``RydbergPulse`` objects
-    pulses = [RydbergPulse(amplitude, phase, detuning, wires)]
+    pulses = [RydbergPulse(amplitude, phase, detuning, None, wires)]
 
     return RydbergHamiltonian(coeffs, observables, pulses=pulses)
 
 
+def rydberg_local_shift(detuning, pattern, wires):
+    r"""Returns a :class:`ParametrizedHamiltonian` representing the action of a local shifting laser
+    with a given temporal and spatial detuning.
+
+    .. math::
+        -\delta(t) \sum_{i \in \text{wires}} h_i \sigma_i^z
+
+    where :math:`\delta` is the temporal detuning, :math:`h_i` is the spatial detuning, or site coefficient
+    between 0 and 1, of the laser, :math:`i` correspond to the wire index, and :math:`\sigma^z` is the Pauli
+    Z matrix. The unit of time for the evolution of this local Rydberg shift term is :math:`\mu \text{s}`. This
+    local shift term can be combined with an interaction and global drive term to create a Hamiltonian describing
+    a driven Rydberg atom system. Multiple shifting terms can be combined by summing them (see example).
+
+    Args:
+        detuning (Union[float, Callable]): float or callable returning the detuning (in MHz) of a
+            laser field
+        pattern (List[float]): list containing the spatial detuning factor between 0 and 1 corresponding to each
+            Rydberg atom being locally driven
+        wires (Union[int, List[int]]): integer or list containing wire values for the Rydberg atoms that
+            the laser field acts on
+
+    Returns:
+        RydbergHamiltonian: a :class:`~.ParametrizedHamiltonian` representing the action of the laser field
```

Comment:
I think it is a bit unusual to refer to the parent class of the (already indicated) returned type, but that might be me :)